### DISCUSSION TITLE
Unable to set SyncThing LXC `passwd`

### DISCUSSION THREAD

**Discussion by msinfo32github** [tteck/Proxmox#1662] [Help Needed]
I am unable to set a password for `root` in an LXC created by the SyncThing LXC script. 

When attempting to run `passwd` and setting a new password:

```
root@syncthing:~# passwd
New password: 
Retype new password: 
Sorry, passwords do not match.
passwd: Authentication token manipulation error
passwd: password unchanged
```

When running `sudo passwd root` and setting a new password:

```
root@syncthing:~# sudo passwd root
sudo: /etc/sudo.conf is owned by uid 100000, should be 0
sudo: /usr/bin/sudo must be owned by uid 0 and have the setuid bit set
```

I have:
- attempted the set root password section of [this](https://github.com/tteck/Proxmox/discussions/324)
- have seen [this](https://github.com/tteck/Proxmox/discussions/131)

Is there something I am doing wrong?

**Comment by normous** [comment]
I am experiencing the exact same thing on a Jellyfin container.

**Comment by Mellowlynx** [comment]
Just did a clean install and works fine.

![image](https://github.com/user-attachments/assets/f909612b-4bcd-4025-ba5c-ba554346a18f)
